### PR TITLE
Hotfix/update jenkinsfile

### PR DESCRIPTION
**Pull request by sebastian-aranda**
This PR makes a small change to set the `main` branch (previously named as `master`) on the Jenkins pipeline.

### CODE REVIEW

**Review by fdelcampo — COMMENTED** [review]
In the file of Jenkinsfile, in the stage of "Run test", I think need some definitions, but maybe for nature of the commander, not need. Confirm please.

**Review by fdelcampo — APPROVED** [review]
It's OK changes

**Comment by sebastian-aranda** [comment]
> In the file of Jenkinsfile, in the stage of "Run test", I think need some definitions, but maybe for nature of the commander, not need. Confirm please.

Good question. This changes were previously added to https://github.com/lsst-ts/LOVE-commander/pull/41/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8. We should wait until that changes are merged to restore the tests on the pipeline.

**Pull request merged**
